src/Factories.WebApi.BLL/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+USER app
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+COPY ["src/Factories.WebApi.BLL/Factories.WebApi.BLL.csproj", "src/Factories.WebApi.BLL/"]
+COPY ["src/Factories.WebApi.DAL/Factories.WebApi.DAL.csproj", "src/Factories.WebApi.DAL/"]
+RUN dotnet restore "./src/Factories.WebApi.BLL/Factories.WebApi.BLL.csproj"
```

Comment:
плюсик за отдельное копирование csproj и нугет рестор, для ускорения билда при кэшировании этих слоёв. рекомендую также почитать [статью](https://andrewlock.net/optimising-asp-net-core-apps-in-docker-avoiding-manually-copying-csproj-files-part-2/#the-new-improved-solution), я в своём проекте использовал немного более универсальные команды, чтобы не добавлять все csproj по отдельности:

```
# Copy only csproj files first before nuget restore to force caching nuget restore layers
# https://andrewlock.net/optimising-asp-net-core-apps-in-docker-avoiding-manually-copying-csproj-files-part-2/#the-new-improved-solution

COPY src/*/*.csproj ./
RUN for file in $(ls *.csproj); do mkdir -p src/${file%.*}/ && mv $file src/${file%.*}/; done

COPY tests/*/*.csproj ./
RUN for file in $(ls *.csproj); do mkdir -p tests/${file%.*}/ && mv $file tests/${file%.*}/; done

# Restore NuGet packages for entire solution, so that this layer could be reused by any of projects from this solution
RUN dotnet restore

# Copy the source
COPY src/ ./src/

# Build the application
RUN dotnet build src/WorkerService/WorkerService.csproj -c Release --no-restore
```